python/qemu/ot/otp/descriptor.py

```diff
@@ -122,11 +123,32 @@ def save(self, kind: str, hjname: str, scriptname: str, cfp: TextIO) \
                     if isinstance(attr_val, bool):
                         attr_val = str(attr_val).lower()
                     print(f'        .{attr_name} = {attr_val},', file=cfp)
-            print(f'    }},', file=cfp)  # noqa: F541
+            print('    },', file=cfp)
         print('};', file=cfp)
         print('', file=cfp)
-        print('#define OTP_PART_COUNT ARRAY_SIZE(OtOTPPartDescs)', file=cfp)
+        print('#define OTP_PART_COUNT ARRAY_SIZE(OT_OTP_PART_DESCS)', file=cfp)
         print(file=cfp)
+        key_seeds: list[str, str, int, int] = []
```

Comment:
Typing nit - [should be](https://docs.python.org/3/library/typing.html#annotating-tuples):

```suggestion
        key_seeds: list[tuple[str, str, int, int]] = []
```
?